src/Spout/Writer/Exception/InvalidBorderStyleException.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Box\Spout\Writer\Exception;
```

Comment:
Flat is better ... But it looks weird just listing the folder with the Border* Exceptions. I'll introduce a Border folder ;)